runbotics/runbotics-desktop/package.json

```diff
@@ -30,6 +30,7 @@
         "winax": "^3.1.5"
     },
     "dependencies": {
+        "@nestjs/axios": "~1.0.0",
```

Comment:
remove this dep, it was causing problems with windows bot installation

Reply:
done